src/SFA.DAS.Tools.Support.Web/Startup.cs

```diff
@@ -76,7 +76,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 var policy = new AuthorizationPolicyBuilder()
                     .RequireAuthenticatedUser()
-                    .RequireClaim("http://service/service", _configuration["RequiredRole"])
+                    .RequireClaim("http://service/service", new string[] { _configuration["RequiredRole"] })
```

Comment:
this needs to be `RequiredRoles` plural and needs to be split as the config value itself can only be a string. Also should update the local appsetting name.